backend/infra/deploy.sh

```
@@ -0,0 +1,39 @@
+  sleep 5
+  if ps -p $PID > /dev/null; then
+    echo "⚠️ 프로세스 $PID 가 아직 종료되지 않음. 강제 종료합니다..."
+    kill -9 $PID
+  fi
+else
+  echo "✅ 80 포트에서 실행 중인 프로세스가 없습니다."
+fi
+
+echo "▶️ 새로운 Spring WAS 실행 중..."
+if [ -f "$JAR_NAME" ]; then
+  sudo nohup java -jar "$JAR_NAME" --spring.profiles.active=prod > /tmp/app.log 2>&1 &
```

Shellcheck (0.10.0)

[warning] sudo doesn't affect redirects. Use ..| sudo tee file

(SC2024)